packages/react-ai/src/components/AIConversation/context/ControlsContext.tsx

```diff
@@ -8,6 +8,7 @@ export interface ControlsContextProps {
     {
       handleSubmit: (e: React.FormEvent<HTMLFormElement>) => void;
       allowAttachments?: boolean;
+      isLoading?: boolean;
```

Comment:
Passing this in so customers can disable the input if the conversation is loading